.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+## Background
+<!-- What would a code reviewer, or a future dev, need to know about this PR in order to understand why this PR is necessary -->
```

Comment:
Should the "why this PR is necessary" part be in the PR or in the connected ticket?

Reply:
IMHO, a reviewer/future dev should be able to get a solid sense of the code's purpose, without having to go out to other linked resources.  My experience has been that people usually won't go read linked resources.

That said, sometimes the issue is much more verbose, or too general.  This gives the developer the opportunity to capture precisely what someone would need to know, based on the specific implementation they are putting forward.

Reply:
Along these lines, perhaps add to the DoD section something like this:
- [ ] Provide links to any originating GH issues

Reply:
I like it.  Be it linking through the ZenHub feature, or dropping the link itself, a good reminder either way.